### ISSUE TITLE
Install procedure no longer works

### ISSUE THREAD

**Issue by egafni** [cpcloud/ipython-autotime#11]
with the latest jupyter, the install instructions no longer works

**Comment by praneet22** [comment]
ERROR:root:Line magic function `%install_ext` not found.

**Comment by miraculixx** [comment]
try

```
# jupyter notebook ipython cell
!pip install git+https://github.com/cpcloud/ipython-autotime
%load_ext autotime
# next cell
print("hello")
=> 
hello
time: 1.59 ms
```

**Comment by cpcloud** [comment]
Fixed in 23963fc
